multiqc/modules/stacks/stacks.py

```diff
@@ -248,14 +248,16 @@ def parse_sumstats(self, file_contents, s_name):
         var_lines = fl[fl.index("# Variant positions")+1:fl.index("# All positions (variant and fixed)")]
         headers = None
         for l in var_lines:
-            if l.startswith("# Pop ID"):
+            if l.startswith("# Pop ID	Private	Num_Indv"):
                 headers = l.split("\t")
+            elif l.startswith("# All positions (variant and fixed)"):
```

Comment:
Instead of this, can we just check the number of elements in `content`?

```suggestion
            elif headers and len(headers) != len(content):
```

Reply:
I don't know how that `elif` statement got in there, it's redundant. See [L248](https://github.com/ewels/MultiQC/pull/1056/files#diff-0ddfc5f28b0365aabb99df67f1a415bdR248) 🤦‍♂ 

The parsing is only going to happen between those two lines to grab that table. I don't find the `# All positions` table terribly interesting, but that's just me.

Reply:
Ah, nice approach 👍